configs/db_demo.yaml

```diff
@@ -0,0 +1,155 @@
+drop_tables: true
+create_tables: true
+print_tables: true
+create_permissions: true
+users:
+  - username: testadmin@cesium-ml.org
+    role_ids:
+      - Super admin
+    tokens:
+      - name: "load_demo_data token"
+        print: true
+        permissions:
+          - "Manage groups"
+          - "Manage sources"
+          - "Upload data"
+          - "Comment"
+          - "Post Taxonomy"
+          - "Manage users"
+          - "System admin"
+  - username: groupadmin@cesium-ml.org
+    role_ids:
+      - Group admin
+  - username: fulluser@cesium-ml.org
+    role_ids:
+      - Full user
+  - username: viewonlyuser@cesium-ml.org
+    role_ids:
+      - View only
+groups:
+  - name: Program A
+    group_admins:
+      - testadmin@cesium-ml.org
+      - groupadmin@cesium-ml.org
+    token: "load_demo_data token"
+    members:
+      - username: viewonlyuser@cesium-ml.org
+        admin: false
+      - username: fulluser@cesium-ml.org
+        admin: false
+    filters:
+      - query_string: "sample_query_string"
+        name: "example filter"
+  - name: Program B
+    group_admins:
+      - testadmin@cesium-ml.org
+      - groupadmin@cesium-ml.org
+    token: "load_demo_data token"
+telescopes:
+  - name: Palomar 1.5m
+    nickname: P60
+    lat: 33.3633675
+    lon: -116.8361345
+    elevation: 1870
+    diameter: 1.5
+    skycam: http://bianca.palomar.caltech.edu/images/allsky/AllSkyCurrentImage.JPG
+    group_names:
+      - Program A
+    token: "load_demo_data token"
+  - name: Nordic Optical Telescope
+    nickname: NOT
+    lat: 28.75
+    lon: 17.88
+    elevation: 2000
+    diameter: 2.56
+    skycam: http://catserver.ing.iac.es/weather/archive/concam/concam_labels.png
+    group_names:
+      - Program A
+    token: "load_demo_data token"
```

Comment:
Why is there a token field in the telescope entries?

Reply:
It's the named token to use to POST the data. In principle multiple tokens can be established and different ones can be used throughout the data import process.